frontend/components/ClubEditPage/AnalyticsCard.tsx

```diff
@@ -71,9 +159,8 @@ function parsePie(obj): PieData {
     output.push({
       angle: item.count,
       label,
-      color: color.rgb().string(),
+      color: colors[colorIndex++].rgb().string(),
```

Comment:
Is there a reason you use `Color(...)` and `.rgb().string()` here instead of just passing in the hex string?

I think that the library should support hex strings as colors.

Reply:
Haha that's a great question. I'll change this now.